packages/truffle-box/lib/utils/unbox.js

```diff
@@ -10,37 +10,33 @@ const inquirer = require("inquirer");
 function verifyURL(url) {
   // Next let's see if the expected repository exists. If it doesn't, ghdownload
   // will fail spectacularly in a way we can't catch, so we have to do it ourselves.
-  return new Promise(function(accept, reject) {
-    var configURL = parseURL(
-      vcsurl(url)
+  return new Promise((accept, reject) => {
+    const configURL = parseURL(
+      `${vcsurl(url)
         .replace("github.com", "raw.githubusercontent.com")
-        .replace(/#.*/, "") + "/master/truffle-box.json"
+        .replace(/#.*/, "")}/master/truffle-box.json`
     );
 
-    var options = {
+    const options = {
       method: "HEAD",
-      uri: "https://" + configURL.host + configURL.path
+      uri: `https://${configURL.host}${configURL.path}`
     };
-    request(options, function(error, r) {
+    request(options, (error, { statusCode }) => {
```

Comment:
Would we consider changing some of these uses of `request` to something that supports promises? Something like `request-promise` perhaps? 
https://www.npmjs.com/package/request#promises--asyncawait

Reply:
Love the variable change by the way haha